python/cudf_polars/cudf_polars/dsl/expressions/aggregation.py

```diff
@@ -184,9 +177,7 @@ def _sum(self, column: Column) -> Column:
         if column.size == 0 or column.null_count == column.size:
             return Column(
                 plc.Column.from_scalar(
-                    plc.interop.from_arrow(
-                        pa.scalar(0, type=plc.interop.to_arrow(self.dtype))
-                    ),
+                    plc.Scalar.from_py(0, self.dtype),
```

Comment:
Weird - This is causing a test failure on the experimental multi-GPU branch

```...
    plc.Scalar.from_py(0, self.dtype),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "scalar.pyx", line 131, in pylibcudf.scalar.Scalar.from_py
  File "/raid/rzamora/conda/envs/rapidsmp-dev/lib/python3.12/functools.py", line 912, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "scalar.pyx", line 273, in pylibcudf.scalar._
TypeError: Cannot convert int to Scalar with dtype FLOAT64
```

Reply:
Was pylibcudf rebuilt on that test branch with this change merged yesterday? https://github.com/rapidsai/cudf/pull/18559

Reply:
>Was pylibcudf rebuilt on that test branch with this change merged yesterday? https://github.com/rapidsai/cudf/pull/18559

Nice - Thanks for the tip!  Looks like that is probably what I need.